### PR TITLE
Add format 0x16 for water-level sensing apps

### PR DESCRIPTION
**Pull request by svelmurugan92**

### CODE REVIEW

File: src/Catena_TxBuffer.h [hunk 1]
```diff
@@ -33,6 +33,8 @@ Revision history:
         #7: Fix error in handling of negative args to 
         TxBuffer_t::put2(int32_t) and TxBuffer_t::put2(int32_t).
 
+   0.8.0 Sat Feb 03 2018 16:48:02 vel
+    Added 0x16 Format.
 */
```
Comment (terrillmoore) [inline src/Catena_TxBuffer.h]: Always need a blank line before trailing `*/`. Please insert.

File: src/Catena_TxBuffer.h [hunk 2]
```diff
@@ -263,6 +266,27 @@ FlagsSensor3 operator|= (FlagsSensor3 &lhs, const FlagsSensor3 &rhs)
...
+        {
+        return FlagsSensor4(uint8_t(lhs) | uint8_t(rhs));
+        };
+
+FlagsSensor4 operator|= (FlagsSensor4 &lhs, const FlagsSensor4 &rhs)
+        {
+        lhs = lhs | rhs;
+        return lhs;
+        };
+
```
Comment (terrillmoore) [inline src/Catena_TxBuffer.h]: This is fine, but looking at this, I wonder if we could use a C++ template somehow so we don't have to repeat all the `operator|` definitions.

File: src/Catena_TxBuffer.h [hunk 3]
```diff
@@ -8,10 +8,10 @@ Module:  Catena_TxBuffer.h
 	namespace McciCatena, class TxBuffer_t;
 
 Version:
-	V0.6.1	Wed Nov 08 2017 22:35:02 tmm	Edit level 3
+	V0.8.0	Sat Feb 03 2018 16:48:02 vel	Edit level 3
```
Comment (terrillmoore) [inline src/Catena_TxBuffer.h]: Edit level *4*.

File: src/Catena_TxBuffer.h [hunk 4]
```diff
@@ -33,6 +33,8 @@ Revision history:
         #7: Fix error in handling of negative args to 
         TxBuffer_t::put2(int32_t) and TxBuffer_t::put2(int32_t).
 
+   0.8.0 Sat Feb 03 2018 16:48:02 vel
+    Added 0x16 Format.
```
Comment (terrillmoore) [inline src/Catena_TxBuffer.h]: indentation of added line 37 looks wrong compared to line 34. Please check

**Comment by terrillmoore** [comment]
@svelmurugan92, you need to push changes to the same branch. per my review. Please let me know when you've done that.

**Comment by svelmurugan92** [comment]
@terrillmoore I pushed the changes. Shall we merge.

**Comment by terrillmoore** [comment]
Vel, also you need to update the files in https://github.com/mcci-catena/Catena-Sketches/tree/master/extra to support format 0x16: `catena-message-0x11-0x15-decoder-node-red.js`,
`catena-message-0x14-0x15-decoder-ttn.js`,
`catena-message-0x14-decoder-node-red.js`,
and `catena-message-0x14-format.md`. That is a separate repo, so needs to be handled there (but those docs & samples need to move to this repo -- more on that in the future).

**Pull request merged**